dbcon/mysql/ha_mcs_sysvars.cpp

```diff
@@ -419,6 +429,16 @@ void set_group_by_handler(THD* thd, bool value)
     THDVAR(thd, group_by_handler) = value;
 }
 
+bool get_select_handler_in_stored_procedures(THD* thd)
+{
+    return ( thd == NULL ) ? false :
```

Comment:
Since the default value of the variable is `true`, we should return back a `true` here for the case when `thd == NULL`